contracts/Staking.sol

```diff
@@ -3,10 +3,14 @@ pragma solidity ^0.6.0;
 
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@nomiclabs/buidler/console.sol";
 
 contract Staking {
     using SafeMath for uint256;
 
+    uint256 constant MULTIPLIER_DECIMALS = 4;
+    uint16 constant BASE_MULTIPLIER = uint16(1 * 10 ** MULTIPLIER_DECIMALS);
```

Comment:
remove MULTIPLIER_DECIMALS maybe? saw that you use only BASE_MULTIPLIER

Reply:
also why not add directly 10000 ?

Reply:
MULTIPLIER_DECIMALS is used throughout the code.

Reply:
not really, it was sed to recompute 10** MULTIPLIER_DECIMALS which is precisely BASE_MULTIPLIER

Reply:
@abancu is right. Removed `MULTIPLIER_DECIMALS`  and replaced every calculation with `BASE_MULTIPLIER`